docs/releasenotes/RELEASE-NOTES-5.0.0.md

```diff
@@ -0,0 +1,67 @@
+# Semantic MediaWiki 5.0.0
+
+Released on November 18, 2024.
+
+## Summary
+
+This release mainly brings support for recent versions of MediaWiki and PHP.
+Anyone using MediaWiki 1.41 or above, or PHP 8.1 or above, is recommended to upgrade.
+
+## Compatibility
+
+* Added support for MediaWiki 1.42
+* Improved compatibility with MediaWiki 1.43
+* Improved compatibility with PHP 8.1 and above
+* Dropped support for MediaWiki older than 1.39
+* Dropped support for PHP older than 8.0
+
+For more detailed information, see the [compatibility matrix](../COMPATIBILITY.md#compatibility).
+
+### Deprecated ResourceLoader modules
+| Deprecated module| Suggestion| Commit
```

Comment:
Are those deprecated or removed?

Reply:
They are hard deprecated so yes they are removed. I'll update the note accordingly.

Reply:
On a side note, should we mark the version as alpha in the notes until it actually releases?

Reply:
> On a side note, should we mark the version as alpha in the notes until it actually releases?


Could do